packages/extras/src/lib/components/controls/TransformControls/TransformControls.svelte

```diff
@@ -22,14 +22,17 @@
   type $$Slots = TransformControlsSlots
 
   export let autoPauseOrbitControls: $$Props['autoPauseOrbitControls'] = true
+  export let autoPauseTrackballControls: $$Props['autoPauseTrackballControls'] = true
   export let object: $$Props['object'] = undefined
 
   const { camera, renderer, invalidate, scene } = useThrelte()
 
-  const { orbitControls } = useControlsContext()
+  const { orbitControls, trackballControls } = useControlsContext()
   const isDragging = writable(false)
   const useAutoPauseOrbitControls = writable(autoPauseOrbitControls) ?? true
```

Comment:
```suggestion
  const useAutoPauseOrbitControls = writable(autoPauseOrbitControls ?? true)
```
I guess this was a preexisting issue 😂

Reply:
lol, patched!